crates/sound/src/vhu_sound.rs

```diff
@@ -410,13 +410,58 @@ impl VhostUserSoundThread {
                     len = desc_response.len();
                 }
 
-                VIRTIO_SND_R_PCM_START | VIRTIO_SND_R_PCM_STOP => {
+                VIRTIO_SND_R_PCM_START => {
                     let pcm_hdr = desc_chain
                         .memory()
                         .read_obj::<VirtioSoundPcmHeader>(desc_request.addr())
                         .map_err(|_| Error::DescriptorReadFailed)?;
-                    let stream_id: usize = u32::from(pcm_hdr.stream_id) as usize;
-                    dbg!("stream_id: {}", stream_id);
+                    let stream_id = u32::from(pcm_hdr.stream_id);
+
+                    let state = audio_backend.get_stream_state(stream_id).unwrap();
```

Comment:
We forgot to handle this error, but I think we can do a patch on top to fix it.

@MatiasVara WDYT?

Reply:
I could create an issue to not forget to deal with the handling of the errors. @stefano-garzarella That would work?